src/RollImage.cpp

```diff
@@ -1066,7 +1078,7 @@ void RollImage::analyzeRawRowPositions(void) {
 	rrp.resize(0);
 
 	std::vector<int>& cch = correctedCentroidHistogram;
-	for (uint i=0; i<cch.size(); i++) {
+	for (int i=0; i<cch.size(); i++) {
```

Comment:
Just curiosity really, but what was the motivation for this?  `i` can't ever be negative, can it?

Reply:
I actually just changed this because it was generating what seems to be an unwarranted red squiggly warning in VS Code (`identifier "uint" is undefined C/C++(20)`) and it was annoying me. And as you point out, `i` can't be negative anyway.